ios/browser/api/url/url_utils.h

```diff
@@ -37,6 +37,13 @@ OBJC_EXPORT
 /// Returns the canonicalized URL's spec.
 @property(readonly) NSString* brave_spec NS_SWIFT_NAME(spec);
 
+/// Returns the host's ETLD+1
+@property(readonly) NSString* brave_ETLDPlusOne NS_SWIFT_NAME(etldPlusOne);
+
+/// Returns the ETLD+1 for the given host.
```

Comment:
Please add comments which say that `lookalikes::GetETLDPlusOne` and `brave_ETLDPlusOne` always ignore Public Suffix List https://chromium.googlesource.com/chromium/src/+/a6b767899dd595d80c59956cd43b23c15f818f16/chrome/browser/lookalikes/lookalike_url_service.h#27. These functions should not be used directly in the code